datasource/src/main/resources/liquibase/changelog/changelog-0.1.xml

```diff
@@ -65,4 +65,21 @@
         </rollback>
     </changeSet>
 
+    <changeSet id="create_run_table" author="manuel_vigueras">
+        <createTable tableName="run">
+            <column name="run_id" type="${uuid.type}"/>
+            <column name="project_id" type="${uuid.type}"/>
+            <column name="project_version" type="${text.type}"/>
+            <column name="status" type="${text.type}"/>
+            <column name="time_start" type="${timestamp.type}"/>
+            <column name="time_end" type="${timestamp.type}"/>
+            <column name="user_id" type="${uuid.type}"/>
+            <column name="results" type="${text.type}"/>
+            <column name="cmwl_workflow_id" type="${text.type}"  defaultValueBoolean="true"/>
+        </createTable>
+        <addPrimaryKey constraintName="run_pk" tableName="run" columnNames="run_id"/>
+        <addForeignKeyConstraint constraintName="fk_run_user" baseTableName="run" baseColumnNames="user_id" referencedTableName="user" referencedColumnNames="user_id"/>
+        <addForeignKeyConstraint constraintName="fk_run_project" baseTableName="run" baseColumnNames="project_id" referencedTableName="project" referencedColumnNames="project_id"/>
+    </changeSet>
```

Comment:
You should also add `not null` constraints for every field that you are consider to be required. In the `Run` class all fields are required (there are no `Option[_]` fields for now), which means this constraint must be added for all of these fields.

Reply:
Changed